packages/debian/download_packages.py

```diff
@@ -111,12 +111,12 @@ def add_external_repositories(salt_version: str) -> None:
             'name': 'saltstack',
             'key': urllib.parse.urljoin(
                 'https://repo.saltstack.com/',
-                'apt/ubuntu/18.04/amd64/{}/SALTSTACK-GPG-KEY.pub'.format(
+                'apt/ubuntu/18.04/amd64/archive/{}/SALTSTACK-GPG-KEY.pub'.format(
```

Comment:
I think i am unable to access the GPG key using an incomplete version prefix such as `2018.3`

We probably need to pass the complete version info `2018.3.4`

Reply:
Something like this works for me: `https://repo.saltstack.com/apt/ubuntu/18.04/amd64/archive/2018.3.4/SALTSTACK-GPG-KEY.pub`